CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## Unreleased
 
 - Deprecates Instabug.start in favour of Instabug.init that takes a configuration object for SDK initialization.
+- Deprecates Instabug.setDebugEnabled, Instabug.setSdkDebugLogsLevel, and APM.setLogLevel in favour of debugLogsLevel property, which can be passed to InstabugConfig while initializing the SDK using Instabug.init.
+- Deprecates the enums: sdkDebugLogsLevel and logLevel in favour of a new enum LogLevel.
```

Comment:
**suggestion (non-blocking):** We can use a little bit better wording here, what do you think?

```suggestion
- Deprecates the enums: sdkDebugLogsLevel and logLevel in favour of the new LogLevel enum.
```

Reply:
@TheBuggedYRN I added these changelog items just to list the changes, but I think we should revisit it once `init` is finalized altogether to compose a clearer changelog before the release.